### PR TITLE
Updating pubspec.yml requirement

### PR DESCRIPTION
**Pull request by MathieuAuclair**
I'm trying to fix the same issue from [here](https://github.com/google/uri.dart/pull/28)

I've updated project requirement to fit with flutter 2.0.0
I might have done something wrong in my PR, this is my first time making a PR in a big project like this, so just tell me what's wrong and I'll change it!

**All test are passing in local**

Thanks you very much!
Have a great day!

### CODE REVIEW

**Comment by kaiquegazola** [comment]
@kevmoo

**Comment by diogoribeiro23** [comment]
@kevmoo @jcollins-g  can someone please accept this?

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,6 @@
 .packages
 .pub
 pubspec.lock
+
+# IntelliJ IDEA
+/.idea
```
Comment (kevmoo) [inline .gitignore]: Please remove this. https://www.dartlang.org/guides/libraries/private-files

**Comment by MathieuAuclair** [comment]
Thanks for your review @kevmoo, if there's anything wrong, I'll fix it :smiley:

**Comment by MathieuAuclair** [comment]
Is there something to do, or you will merge this PR later after checkups?

**Comment by kevmoo** [comment]
published and tagged!

**Pull request merged**